Expensify/App
# Don't allow removing the policy owner from workspace members

### Details
Disable the checkbox and shows a tooltip if the admin tries to remove the owner or themselves from the workspace.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/179467

### Tests / QA

1. In an account that has an existing workspace (or you can create a new one), click on People.
2. Add a couple of extras users to the workspace.
3. Make sure you can select all users to be removed except for the owner of the workspace and yourself.
4. If you hover on web or desktop on the owner or yourself, you should get the tooltip shown in the screenshots.
5. If you test this on a mobile device, a growl should show up instead (as shown in the screenshots)
6. Play with the `Select All` button and make sure it still works, it should select all users except for the above.
7. Test that removing the selected users work as expected and they get removed.

### Tested On


### Screenshots


#### Web
![New Expensify 2021-09-30 17-30-13](https://user-images.githubusercontent.com/6474442/135499859-c6b12e33-49f5-4cba-88c8-a8bb627dc004.png)

#### Mobile Web
![iPhone 12 2021-10-04 15-13-02](https://user-images.githubusercontent.com/6474442/135877734-28b796c9-159e-4489-9aac-46e695a5f61e.png)

#### Desktop
![New Expensify 2021-09-30 17-30-13](https://user-images.githubusercontent.com/6474442/135499878-1f05a896-fdcc-4b5e-a9f8-878b03b8b129.png)

#### iOS
![iPhone 12 2021-10-04 12-16-14](https://user-images.githubusercontent.com/6474442/135825603-ce9f99bf-0cbe-4b63-84b0-81ba2e80c896.png)

#### Android
![Android Emulator - Pixel_5_API_30:5554 2021-10-04 11-36-42](https://user-images.githubusercontent.com/6474442/135820799-84f1f2ca-b169-406e-a34d-4447e94720f9.png)
